docs/mpe.md

```diff
@@ -3,47 +3,51 @@
 
-* Communication: Some agents in some environments can broadcast a message as part of its action (see action space for more details) which will be transmitted to each agent that is allowed to see that message. In simple_crypto, this message is also used to signal that Bob and Eve have reconstructed the message.
+* **Communication**: Some agents in some environments can broadcast a message as a part of its action (see action space for more details) which will be transmitted to each agent that is allowed to see that message. In Simple Crypto, this message is used to signal that Bob and Eve have reconstructed the message.
 
-* Color: Since all agents are rendered as circles, the agents are only identifiable to a human by their color, so the color of the agents is described in most of the environments. The color is not observed by the agents.
+* **Color**: Since all agents are rendered as circles, the agents are only identifiable to a human by their color, so the color of the agents is described in most of the environments. The color is not observed by the agents.
 
-* Distances: The landmarks and agents typically start out uniformly randomly placed from -1. to 1. on the map. This means they are typically around 1-2 units distance apart, so keep that in mind when reasoning about the scale of the rewards, which often depend on distance, and the observation space, which contain relative and absolute positions.
+* **Distances**: The landmarks and agents typically start out uniformly randomly placed from -1 to 1 on the map. This means they are typically around 1-2 units apart. This is important to keep in mind when reasoning about the scale of the rewards (which often depend on distance) and the observation space, which contains relative and absolute positions.
 
 ### Termination
 
-The game terminates after a number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original source code, the default value was 25, not 100.
+The game terminates after the number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original OpenAI source code, the default value was 25, not 100.
 
 ### Observation Space
 
-The observation space of an agent is a vector generally composed of the agent's position and velocity, other agent's relative position and velocity, the landmarks relative positions, the landmark's and agent's types, and communications it received from other agents. The exact form of this is detailed in the environments.
+The observation space of an agent is a vector generally composed of the agent's position and velocity, the other agents' relative positions and velocities, the landmarks' relative positions, the landmarks' and agents' types, and communications received from other agents. The exact form of this is detailed in the environments' documentation.
```

Comment:
Remove the "the" before other agents'. It's plural not singular here.

Reply:
the agents and the agent are both correct phrases? The "the" keeps the format of the phrases in the comma-separated list consistent.

On that note I just realized I need to add one before "communications"

Reply:
I have no idea what you just said. This one may need a 2 minute call.

Reply:
Solution: remove all the "the"s